.github/workflows/publish-release.yaml

```diff
@@ -13,3 +13,5 @@ jobs:
     needs: [test-and-build]
     with:
       isReleaseCandidate: false
+    secrets:
+      githubToken: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
```

Comment:
should this be `GITHUB_TOKEN: ${{ secrets.githubToken }}` too?

Reply:
This is the calling workflow so it needs to use the actual secret name.